OrbitGl/GlCanvas.cpp

```diff
@@ -138,6 +138,13 @@ void GlCanvas::EnableImGui() {
   }
 }
 
+orbit_accessibility::AccessibleInterface* GlCanvas::Accessibility() {
+  if (accessibility_ == nullptr) {
+    accessibility_ = CreateAccessibilityInterface();
```

Comment:
Now that CreateAccessibilityInterface() is private, if it's not called elsewhere, should we remove it and inline the code here instead?

Reply:
It's overridden in `CaptureWindow`. That's the whole idea of this method - it's a template method that deriving classes can override to construct specialized instances of the accessibility interface.

Reply:
Ah yes, cool.